src/SignalR/clients/csharp/Client.Core/src/HubConnectionBuilder.cs

```diff
@@ -80,6 +80,9 @@ public override string ToString()
         }
 
         // Prevents from being displayed in intellisense
+        /// <summary>
+        /// Gets the <see cref="Type"/> of the current instance.
+        /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public new Type GetType()
```

Comment:
Maybe a little off topic, but why do these methods exist? I'm also referring to the GetHashCode, Equals and ToString overrides.

Reply:
No idea. My best guess is to add the `EditorBrowsable` attribute.

Reply:
Yup